test/unit/uk/gov/hmrc/tariffclassificationfrontend/connector/Payloads.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package unit.uk.gov.hmrc.tariffclassificationfrontend.connector
+
+
+import java.time.LocalDate
+
+import play.api.libs.json.{Json, OWrites, Writes}
+import uk.gov.hmrc.tariffclassificationfrontend.models.Case
+
+object Payloads {
+
+  implicit val writes: OWrites[Case] = Json.writes[Case]
+
+  val GATEWAY_CASES: String = jsonOf(Seq(Case(
```

Comment:
I think the Scala community prefers camel case for `val`s

Reply:
Done